ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -339,7 +339,7 @@ tests:
 - as: e2e-metal-ipi-sdn
   cluster: build05
   optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|alibabacloud|aws|azure|azurestack|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: (baremetal|metal3-io)
```

Comment:
These are paths like terraform/providers/ironic/vendor/github.com/openshift-metal3/terraform-provider-ironic/ironic/resource_ironic_deployment.go which don't match this, so potentially the CI won't be triggered on terraform updates. Do we care? Maybe use baremetal|metal3|ironic?

Reply:
will update in a follow up pr right after this one merges

Reply:
Needed a rebase, so I incorporated these changes.